src/main/java/graphql/execution/batched/FetchedValues.java

```diff
@@ -11,24 +12,26 @@
 public class FetchedValues {
 
     private final List<FetchedValue> fetchedValues;
-    private final ExecutionStepInfo executionStepInfo;
-    private final ResultPath path;
+    private final DataFetchingEnvironment dataFetchingEnvironment;
 
-    public FetchedValues(List<FetchedValue> fetchedValues, ExecutionStepInfo executionStepInfo, ResultPath path) {
```

Comment:
I don't feel like this is better - it had just the right variables and collapsing them back to its source DFE does not make things clearer